tests/token/erc20/test_ERC20Upgradeable.py

```diff
@@ -17,145 +17,131 @@
 DECIMALS = 18
 
 
-@pytest.fixture(scope='module')
-def contract_classes():
-    account_cls = Account.get_class
-    token_cls = get_contract_class('ERC20Upgradeable')
-    proxy_cls = get_contract_class('Proxy')
-
-    return account_cls, token_cls, proxy_cls
-
-
-@pytest.fixture(scope='module')
-async def token_init(contract_classes):
-    account_cls, token_cls, proxy_cls = contract_classes
-    starknet = await State.init()
-    account1 = await Account.deploy(signer.public_key)
-    account2 = await Account.deploy(signer.public_key)
-    token_v1 = await starknet.declare(
-        contract_class=token_cls,
-    )
-    token_v2 = await starknet.declare(
-        contract_class=token_cls,
-    )
-    proxy = await starknet.deploy(
-        contract_class=proxy_cls,
-        constructor_calldata=[token_v1.class_hash]
-    )
-    return (
-        starknet.state,
-        account1,
-        account2,
-        token_v1,
-        token_v2,
-        proxy
-    )
-
-
-@pytest.fixture
-def token_factory(contract_classes, token_init):
-    account_cls, _, proxy_cls = contract_classes
-    state, account1, account2, token_v1, token_v2, proxy = token_init
-    _state = state.copy()
-    account1 = cached_contract(_state, account_cls, account1)
-    account2 = cached_contract(_state, account_cls, account2)
-    proxy = cached_contract(_state, proxy_cls, proxy)
-
-    return account1, account2, proxy, token_v1, token_v2
-
-
-@pytest.fixture
-async def after_initializer(token_factory):
-    admin, other, proxy, token_v1, token_v2 = token_factory
-
-    # initialize
-    await signer.send_transaction(
-        admin, proxy.contract_address, 'initializer', [
+class TestERC20Upgradeable:
```

Comment:
Allow marking classes instead of every test (for example, to mark with "only" while developing). Also, for organization, trying to follow mocha pattern for nesting tests (contract and describe).

Reply:
the former doesn't sound bad (if documented properly), but I'd be wary of following other tooling's patterns, let alone other languages.

Reply:
Ahh gotcha. I was thinking tests are already grouped well IMO within the modules, but I think marking would be a helpful